script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -1,32 +1,32 @@
 <Page1>
 I can't go down like this.
-The ground below is completely filled with blue flames.
+The floor of the pit is completely covered in blue flames.
 <Page2>
 A tall vampire stands at the center.
 Vlov Arkhangel.
-He stands in the middle of the crater without a single scratch.
+He was right at the center of that explosion, but there's not a scratch on him.//あの爆撃のただ中にいて、傷一つ負っていない。
 By human standards, he's practically an immortal monster.
-And then...
+And then―――
 <Page3>
 There is a white figure toying with the man from the front.
-The beautiful hunter darts through the garden of flickering flames and debris.
+A beautiful huntress darting through the garden of flickering flames and debris.
 <Page4>
-Of course, that white figure belongs to Arcueid.
+Of course, that white figure is Arcueid.
 She leaps across the river of flames and closes in on Vlov.
 <Page5>
-He has a crude and deadly weapon gripped in his hand.
-Although it has the appearance of a long sword, the blade is that of a billhook.
+He has a crude yet deadly weapon gripped in his hand.
+Although it has the appearance of a long sword, the blade is that of a machete.
```

Comment:
Yeah, I hadn't encountered billhook before but I see where it came from - 
```
鉈 (なた) (n) nata hatchet; traditional Japanese edged tool with short handle and wide blade; 
鉈鎌 (なたがま) (n) (uk) billhook; fascine knife; sickle-shaped tool with a sharp inner edge, halfway between a knife and an axe; 
```
my main reason for changing it was that I felt it emphasized the _hook_ far too much, which isn't really a property of a plain 鉈 / Vlov's sword. Image searching 鉈 it looks to me like a boxier machete (googling knife names, the 'Tapanga'? I could see Danny Trejo with one at least). Romanizing it directly might be a bit much, I don't think it's a common enough loan word.

Reply:
I don't really have an opinion on this one. I had never come across the word billhook before but it does seem to be more accurate. I was thinking of changing it too but I thought that it sounded more exotic, which might be a connotation that we want. On the flipside, machete is a word that practically anyone reading would know of.

Reply:
Well, I would keep machete, rather than the specific _version_ of machete. It just seems like a better match for this shape:
![2021-12-14-212322_749x853_scrot](https://user-images.githubusercontent.com/2476799/146111392-fd784e44-a159-4faa-ac4a-8f6c299a2cf7.png)

Generic machete results:
![2021-12-14-212522_1571x830_scrot](https://user-images.githubusercontent.com/2476799/146111583-62f33903-1361-4f98-acb6-b04ce9e747b2.png)

whereas a billhook is, well, very _hooked_, which doesn't really match the CGs I think.
![2021-12-14-212403_1568x827_scrot](https://user-images.githubusercontent.com/2476799/146111479-ef1ccd83-88b2-439e-b327-454160abc9d5.png)

I guess the nata is also sorta like a big meat cleaver if that works better for people?

Reply:
machete is fine for me

Reply:
I personally prefer machete as well. Reading machete gives more of a sense of danger than billhook.